wallet/models.py

```diff
@@ -5,10 +5,13 @@
 
 WALLET_TYPE = [("Visa", "visa"), ("Mastercard", "mastercard")]
 CURRENCY = [("USD", "dollar"), ("EUR", "euro"), ("RUB", "ruble")]
+MAX_WALLETS = 5
+BANK_BONUS_USD_EUR = 3
+BANK_BONUS_RUB = 100
 
 
 class Wallet(models.Model):
```

Comment:
please add` __str__` method to the class

Reply:
Done